core/domain/collection_jobs_one_off.py

```diff
@@ -78,10 +80,15 @@ def map(item):
                 feconf.MIGRATION_BOT_USERNAME, item.id, commit_cmds,
                 'Update collection schema version to %d.' % (
                     feconf.CURRENT_COLLECTION_SCHEMA_VERSION))
-            yield (
-                CollectionMigrationJob._MIGRATED_KEY,
-                'Collection successfully migrated.')
+            yield (CollectionMigrationOneOffJob._MIGRATED_KEY, 1)
 
     @staticmethod
     def reduce(key, values):
-        yield (key, values)
+        if key == CollectionMigrationOneOffJob._DELETED_KEY:
+            yield (key, ['Encountered %d deleted collections.' % (
+                sum(ast.literal_eval(v) for v in values))])
```

Comment:
Could you please add backend tests, here and elsewhere, for this new code?

Reply:
I'm not sure if I understand what the test is expected to be. Can you point me to any other similar tests in the repo?

Reply:
It would be similar to existing tests in collection_jobs_one_off_test.py (and other one-off test files). In this case, you could set up a collection, delete it, run the job, and then check that the output of the job is as expected. (Note that the core job classes are defined in core/jobs.py.)

Reply:
fixed.